### PR TITLE
holo-files: Fix subtle bug where it didn't require `--force` when it should

### PR DESCRIPTION
**Pull request by LukeShu**
I was working on implementing patch files (#5), when I noticed the

    base = current

line, which should obviously be

    base = tmp

But if you fix the bug without writing a test, is it really fixed?  So then
I went about trying to figure out how to trigger misbehavior based on that
mistake.  In doing so, I found a second bug, when my test very clearly
should have required `--force`, but it didn't.

The fixes to both bugs are required to make the added test pass.

### CODE REVIEW

**Review by majewsky — APPROVED** [review]
Very nice. I love how the testcases cover increasingly obscure scenarios over time. :)

Just one thing, though: Can you please update the `CHANGELOG.md`? Since there are no notes for the next release yet, you should add a new section at the top:

```
# v2.2.1 (TBD)
```

**Comment by majewsky** [comment]
When you have added the changelog entry, feel free to merge this yourself. I've given you write access since I trust you to work on fixes and documentation improvements individually. If you need another pair of eyes for some change, you're of course still welcome to use PRs.

**Comment by LukeShu** [comment]
Sure thing! Thanks for access!

**Comment by LukeShu** [comment]
I've added a changelog entry (and improved the commit message), but I don't seem to have write access.

**Comment by LukeShu** [comment]
Whoops, I just had to accept the invite to the holocm organization.

**Pull request merged**
